### PR TITLE
Trimaker - speed profile

### PR DESCRIPTION
**Pull request by MatiGabaglio16**
Update for the Trimaker vendor bundle. The new version includes a profile for high speed

### CODE REVIEW

File: live/Trimaker/1.1.0.ini [hunk 1]
```diff
@@ -0,0 +1,407 @@
+# Print profiles for the Trimaker printers.
+
+[vendor]
+# Vendor name will be shown by the Config Wizard.
+name = Trimaker
+# Configuration version of this file. Config file will only be installed, if the config_version differs.
+# This means, the server may force the PrusaSlicer configuration to be downgraded.
+config_version = 1.1.0
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:8]: Please change the version to 1.0.1.

File: live/Trimaker/1.1.0.ini [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+[print:0.2 Layer - 0,4 Nozzle]
+inherits = *0.20mm*
+# alias = 0.20mm NORMAL 0,4
+compatible_printers_condition = nozzle_diameter[0]==0.4
+
+[print:0.2 Layer - 0,4 Nozzle - Speed]
+inherits = *0.20mm*
+# alias = 0.20mm NORMAL 0,4
+default_acceleration = 1500
+machine_max_acceleration_x = 1500
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:227]: Please remove this incorrect key.

File: live/Trimaker/1.1.0.ini [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+machine_max_acceleration_y = 1500
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:228]: Please remove this incorrect key.

File: live/Trimaker/1.1.0.ini [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+machine_max_acceleration_extruding = 1500
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:229]: Please remove this incorrect key.

File: live/Trimaker/1.1.0.ini [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+machine_max_jerk_x = 15
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:230]: Please remove this incorrect key.

File: live/Trimaker/1.1.0.ini [hunk 6]
```diff
@@ -0,0 +1,407 @@
...
+machine_max_jerk_y = 15
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:231]: Please remove this incorrect key.

File: live/Trimaker/1.1.0.ini [hunk 7]
```diff
@@ -0,0 +1,407 @@
...
+slowdown_below_layer_time = 0
```
Comment (rtyr) [inline live/Trimaker/1.1.0.ini:232]: Please remove this incorrect key.

File: live/Trimaker/index.idx [hunk 8]
```diff
@@ -1,2 +1,3 @@
 min_slic3r_version = 2.6.0-beta2
+1.1.0 Speed profile has been added
```
Comment (rtyr) [inline live/Trimaker/index.idx:2]: Please change the version to 1.0.1.

**Review by rtyr — CHANGES_REQUESTED** [review]
Please fix the things mentioned in the review.

Summary:
- change the version to 1.0.1 in both files (semantic versioning)
- remove incorrect keys (machine limits and slowdown does not belong to print profile)

**Comment by rtyr** [comment]
version number in index.idx have to be also changed, otherwise it should be ok to merge

**Comment by MatiGabaglio16** [comment]
Hello, why are this changes not visible in prusa slicer yet?

**Comment by rtyr** [comment]
The update is available from the moment this was merged, so roughly 2 months. I can see it in the slicer.

**Pull request merged**